chan_handler.go

```diff
@@ -1,10 +1,12 @@
-// Copyright 2013-2016 Apcera Inc. All rights reserved.
+// Copyright 2013-2017 Apcera Inc. All rights reserved.
 
 package graft
 
 // ChanHandler is a convenience handler when a user wants to simply use
 // channels for the async handling of errors and state changes.
 type ChanHandler struct {
+	LogPositionHandler
```

Comment:
Apply to everything else in the PR: Is `Log` a good choice here? It may imply log replication, which as you know is not done in graft. Would something about vote or state be more appropriate?